--- coolkey-1.1.0/configure.in.orig	2007-02-16 20:50:50.000000000 +0100
+++ coolkey-1.1.0/configure.in	2014-08-22 18:15:55.079051272 +0200
@@ -124,9 +124,9 @@
 if test $WINDOWS -ne 1; then
   PKG_CHECK_MODULES(NSS, nss, true, [ AC_MSG_ERROR(could not find NSS Crypto libraries) ])
 fi
-  enable_pk11install = "yes"
+  enable_pk11install="yes"
 else
-  enable_pk11install = "no"
+  enable_pk11install="no"
   AC_MSG_WARN([skipping pk11install])
 fi
 
